modules/logging-elastic-to-loki-store.adoc

```diff
@@ -0,0 +1,371 @@
+apiVersion: logging.openshift.io/v1
+kind: ClusterLogging
+metadata:
+  name: instance
+  namespace: openshift-logging
+spec:
+  managementState: Managed
+  logStore:
+    type: lokistack
+    lokistack:
+      name: logging-loki
+  collection:
+    logs:
+      type: fluentd
+      fluentd: {}
+  visualization:
+    type: kibana
+    kibana:
+      replicas: 1
+EOF
+----
+
+. Re-instantiate Kibana Resource
+
+In the previous step, removing the `visualization` field prompted the operator to remove the `Kibana` resource. Re-instantiate the `Kibana` resource using the backup created earlier.
+
+[source,bash]
+----
+$ oc -n openshift-logging apply -f /tmp/cr-kibana.yaml
+----
+
+. Enable the Console View Plugin
+
+Enable the console view plugin to view the logs integrated from the OpenShift Console (Observe > Logs).
+
+[source,bash]
+----
+$ oc patch consoles.operator.openshift.io cluster --type=merge --patch '{ "spec": { "plugins": ["logging-view-plugin"] } }'
+----
+
+== Delete the Elasticsearch Stack
+
+Once the retention period for logs stored in Elasticsearch expires and no more logs are visible in Kibana, remove the old stack to release resources.
+
+=== Step 1: Delete Elasticsearch and Kibana Resources
+
+[source,bash]
+----
+$ oc -n openshift-logging delete kibana/kibana elasticsearch/elasticsearch
+----
+
+===
+
+ Step 2: Delete the PVCs Used by Elasticsearch Instances
+
+[source,bash]
+----
+$ oc delete -n openshift-logging pvc -l logging-cluster=elasticsearch
+----
+----
+$ oc -n openshift-logging get elasticsearch elasticsearch -o yaml \
+    | yq 'del(.metadata.resourceVersion) | del(.metadata.uid)'  \
+    | yq 'del(.metadata.generation) | del(.metadata.creationTimestamp)'  \
+    | yq 'del(.metadata.selfLink) | del(.status)' > /tmp/cr-elasticsearch.yaml
+----
+
+For Kibana:
+
+[source,bash]
+----
+$ oc -n openshift-logging get kibana kibana -o yaml \
+    | yq 'del(.metadata.resourceVersion) | del(.metadata.uid)'  \
+    | yq 'del(.metadata.generation) | del(.metadata.creationTimestamp)'  \
+    | yq 'del(.metadata.selfLink) | del(.status)' > /tmp/cr-kibana.yaml
+----
+
+== Switching to LokiStack
+
+. Switch Log Storage to LokiStack
+The following manifest will apply several changes to the `ClusterLogging` resource:
+* Re-instate the management state to `Managed`.
+* Switch the `logStore` spec from `elasticsearch` to `lokistack`, restarting the collector pods to start forwarding logs to `lokistack`.
+* Remove the `visualization` spec, prompting the cluster-logging-operator to install the `logging-view-plugin` for observing `lokistack` logs in the OpenShift Console.
+* If the collection type is not `fluentd`, replace it with `vector`.
+
+[source,yaml]
+----
+$ cat << EOF |oc replace -f -
+apiVersion: logging.openshift.io/v1
+kind: ClusterLogging
+metadata:
+  name: instance
+  namespace: openshift-logging
+spec:
+  managementState: Managed
+  logStore:
+    type: lokistack
+    lokistack:
+      name: logging-loki
+  collection:
```

Comment:
Old datamodel is in this example with collection.logs.type. It should be:
```
  collection:
    type: fluentd 
    fluentd: {}
```
Or as Fluentd is not in RHOL 6.0, probably the best approach is to use Vector as collector:
```
  collection:
    type: vector
```